config/initializers/blog.rb

```diff
@@ -0,0 +1,23 @@
+#See http://www.sitepoint.com/jekyll-rails/
+
+Rails.application.config.after_initialize do
+  Rails.logger = Logger.new(STDOUT)
+  begin
+    #make a spot for the site
+    dest = Rails.root.join('public/blog')
+
+    #generate the site
+    Jekyll::Site.new(
+    Jekyll.configuration({
+      "config" => Rails.root.join('config', 'jekyll.yml'),
+      "source" => Rails.root.join('blog').to_s,
+      "destination" => dest.to_s
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.